docs/architecture/devops-for-aspnet-developers/actions-build.md

```diff
@@ -0,0 +1,193 @@
+
+To explore how you can move code to the cloud, you'll build a GitHub Actions workflow file for the Simple Feed Reader app you've already deployed to Azure App Service.
+
+In this article, you will:
+> [!div class="checklist"]
+
+> * Learn the basic structure of a GitHub Action workflow YAML file.
+> * Use a template to create a basic build workflow that builds the .NET app and executes unit tests.
+> * Publish the compiled app so that it's ready for deployment.
+
+## Workflow structure
+
+Workflows are captured in YAML files and all have the following things:
+
+- a `name`
+- a trigger, defined by an `on` section
+- one or more `job` sections, composed of one or more `steps`
+- optional attributes such as `environment` variables
+
+Jobs are run on _runners_. You can use _hosted runners_, which are spun up by GitHub during the workflow and then thrown away. Hosted runners are great because you don't have to maintain your own build infrastructure. For workflows that require a specific build environment, or for running workflows on a private network, you can also use _private_ runners. To create a private runner, you can install the runner on any machine that supports .NET.
+
+Each `job` will specify what runner GitHub should use to execute the `steps`. You can also specify dependencies between jobs using the `needs` attribute. Deployment jobs can also specify an `environment` to target.
+
+`Steps` can be as easy as inline commands or they can be Actions. Most CI workflows will have a combination of `run` steps (for executing scripts) and Actions. Actions are pulled into the workflow by referencing the Action repository (and optionally a tag or commit hash for specific versions) and specifying any parameters using the `with` keyword.
+
+> [!TIP]
+> You can read more about GitHub Actions YAML syntax [here](https://docs.github.com/actions/reference/workflow-syntax-for-github-actions).
+
+## Create a basic build workflow
+
+One important principle of effective DevOps is to build once, deploy many times. You'll start by creating a workflow to build a basic .NET app. In the next step, you'll publish the output to prepare for deployment.
+
+1. Navigate to your GitHub repositorysitory and select the **Actions** tab.
+1. GitHub detects that there's .NET code in the repositorysitory and suggests a .NET workflow template. Select **Set up this workflow** to create a new YAML workflow file:
+
+    ![Creating a new workflow](./media/actions/build/new-action.jpg)
+    **Figure 1**: Creating a new workflow.
+
+1. Commit the file onto the `main` branch. Since you have defined a trigger condition for *commits to main*, this commit should trigger the workflow to run.
```

Comment:
```suggestion
1. Commit the file onto the `main` branch. Since you've defined a trigger condition for *commits to main*, this commit should trigger the workflow to run.
```